scripts/2.mutation-classifier.py

```diff
@@ -326,19 +326,45 @@ def get_threshold_metrics(y_true, y_pred):
 pd.crosstab(coef_df.feature_set, np.sign(coef_df.weight).rename('coefficient_sign'))
 
 
+# ### Top coefficients for covariates model
+
 # In[21]:
 
-# Top standardized coefficients
-(coef_df
-    .query("feature_set == 'full'")
-    .head(10)
-)
+coef_df.query("feature_set == 'covariates'").head(10)
 
 
-# ## Investigate the predictions
+# ### Top coefficients for full model
 
 # In[22]:
 
+coef_df.query("feature_set == 'full'").head(10)
+
+
+# ### Top coefficients for individual _genes_ for expressions model
```

Comment:
Aren't we equally / more interested in the coefficients from the `full` model?

I think it would be really useful to show the gene coefficients from both models with expression in the same table. You could sort by abs coefficient of the full model, but the comparison between coefficients will be nice.